tests/conversion/viewconsumable.js

```diff
@@ -3,21 +3,29 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+import ViewDocument from '../../src/view/document';
 import ViewElement from '../../src/view/element';
 import ViewText from '../../src/view/text';
 import ViewDocumentFragment from '../../src/view/documentfragment';
 import ViewConsumable from '../../src/conversion/viewconsumable';
-import StylesMap, { StylesProcessor } from '../../src/view/stylesmap';
 import { addBorderRules } from '../../src/view/styles/border';
 import { addMarginRules } from '../../src/view/styles/margin';
 import { addPaddingRules } from '../../src/view/styles/padding';
+import { StylesProcessor } from '../../src/view/stylesmap';
 
 describe( 'ViewConsumable', () => {
-	let viewConsumable, el;
+	let viewConsumable, el, viewDocument, stylesProcessor;
 
 	beforeEach( () => {
+		stylesProcessor = new StylesProcessor();
```

Comment:
This should be internal to `beforeEach()`. The global `stylesProcessor` is not used elsewhere.